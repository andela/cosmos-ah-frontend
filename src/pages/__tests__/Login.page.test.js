// import React from 'react';
// import { BrowserRouter } from 'react-router-dom';
// import { render, cleanup } from 'react-testing-library';
// import Login from '../Login';

// afterEach(cleanup);

// test('Renders <Login />', () => {
//   const LoginComponent = render(<BrowserRouter><Login /></BrowserRouter>);
//   expect(LoginComponent).toBeTruthy();
//   expect(LoginComponent).toMatchSnapshot();
// });

// test('Renders <Login /> form', () => {
//   const { getByPlaceholderText } = render(<BrowserRouter><Login /></BrowserRouter>);
//   expect(getByPlaceholderText('Email Address')).toBeTruthy();
//   expect(getByPlaceholderText('Password')).toBeTruthy();
// });


// test('render <Login />  Login Header', () => {
//   const LoginHeader = render(<BrowserRouter><Login /></BrowserRouter>);
//   const HeaderText = LoginHeader.getByText('Login');
//   expect(HeaderText.textContent).toBe('Login');
// });
