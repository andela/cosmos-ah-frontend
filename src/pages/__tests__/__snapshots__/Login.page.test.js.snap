// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <Login /> 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bwzfXH iYberu"
      >
        <div
          class="sc-htpNat iSSvoI"
        >
          <div>
            <a
              href="/"
            >
              <img
                src="test-file-stub"
                width="200"
              />
            </a>
          </div>
        </div>
        <div
          class="sc-bZQynM bYmjcT"
        >
          <div
            class="sc-bxivhb dAJZi"
          >
            <div
              class="column"
            >
              <div
                class="sc-bZQynM bYmjcT"
              >
                <h1
                  class="ui blue huge header"
                >
                  Login
                </h1>
              </div>
              <div
                class="sc-bdVaJa dSNtGu"
              >
                <div
                  class="ui centered grid"
                >
                  <button
                    class="ui blue massive basic circular button"
                  >
                    <img
                      src="test-file-stub"
                    />
                  </button>
                  <button
                    class="ui red massive basic circular button"
                  >
                    <img
                      src="test-file-stub"
                    />
                  </button>
                  <button
                    class="ui blue massive basic circular button"
                  >
                    <img
                      src="test-file-stub"
                    />
                  </button>
                  <button
                    class="ui blue massive basic circular button"
                  >
                    <img
                      src="test-file-stub"
                    />
                  </button>
                </div>
              </div>
              <div
                class="ui blue header"
              >
                <h4>
                  or login using your email address
                </h4>
              </div>
              <form
                class="ui form"
              >
                <div
                  class="field"
                >
                  <div
                    class="ui big left icon input"
                  >
                    <input
                      placeholder="Email Address"
                      type="text"
                    />
                    <i
                      aria-hidden="true"
                      class="blue envelope outline icon"
                    />
                  </div>
                </div>
                <div
                  class="field"
                >
                  <div
                    class="ui big left icon input"
                  >
                    <input
                      placeholder="Password"
                      type="password"
                    />
                    <i
                      aria-hidden="true"
                      class="blue lock icon"
                    />
                  </div>
                </div>
                <p
                  class="sc-EHOje dhYIww"
                >
                  <a
                    href="/forgot-password"
                  >
                    Forgot password?
                  </a>
                </p>
                <div>
                  <button
                    class="ui blue big fluid button"
                  >
                    Continue
                  </button>
                </div>
              </form>
              <div
                class="sc-ifAKCX bdgWuN"
              >
                <div
                  class="ui centered grid"
                >
                  <div
                    class="sc-bZQynM bYmjcT"
                  >
                    <div
                      class="ui header"
                    >
                      <h3>
                        Don't have an account yet? 
                        <a
                          href="/signup"
                        >
                          Create One
                        </a>
                      </h3>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bwzfXH iYberu"
    >
      <div
        class="sc-htpNat iSSvoI"
      >
        <div>
          <a
            href="/"
          >
            <img
              src="test-file-stub"
              width="200"
            />
          </a>
        </div>
      </div>
      <div
        class="sc-bZQynM bYmjcT"
      >
        <div
          class="sc-bxivhb dAJZi"
        >
          <div
            class="column"
          >
            <div
              class="sc-bZQynM bYmjcT"
            >
              <h1
                class="ui blue huge header"
              >
                Login
              </h1>
            </div>
            <div
              class="sc-bdVaJa dSNtGu"
            >
              <div
                class="ui centered grid"
              >
                <button
                  class="ui blue massive basic circular button"
                >
                  <img
                    src="test-file-stub"
                  />
                </button>
                <button
                  class="ui red massive basic circular button"
                >
                  <img
                    src="test-file-stub"
                  />
                </button>
                <button
                  class="ui blue massive basic circular button"
                >
                  <img
                    src="test-file-stub"
                  />
                </button>
                <button
                  class="ui blue massive basic circular button"
                >
                  <img
                    src="test-file-stub"
                  />
                </button>
              </div>
            </div>
            <div
              class="ui blue header"
            >
              <h4>
                or login using your email address
              </h4>
            </div>
            <form
              class="ui form"
            >
              <div
                class="field"
              >
                <div
                  class="ui big left icon input"
                >
                  <input
                    placeholder="Email Address"
                    type="text"
                  />
                  <i
                    aria-hidden="true"
                    class="blue envelope outline icon"
                  />
                </div>
              </div>
              <div
                class="field"
              >
                <div
                  class="ui big left icon input"
                >
                  <input
                    placeholder="Password"
                    type="password"
                  />
                  <i
                    aria-hidden="true"
                    class="blue lock icon"
                  />
                </div>
              </div>
              <p
                class="sc-EHOje dhYIww"
              >
                <a
                  href="/forgot-password"
                >
                  Forgot password?
                </a>
              </p>
              <div>
                <button
                  class="ui blue big fluid button"
                >
                  Continue
                </button>
              </div>
            </form>
            <div
              class="sc-ifAKCX bdgWuN"
            >
              <div
                class="ui centered grid"
              >
                <div
                  class="sc-bZQynM bYmjcT"
                >
                  <div
                    class="ui header"
                  >
                    <h3>
                      Don't have an account yet? 
                      <a
                        href="/signup"
                      >
                        Create One
                      </a>
                    </h3>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
